src/plugins/mer/meroptionswidget.cpp

```diff
@@ -151,8 +151,13 @@ void MerOptionsWidget::store()
 
     disconnect(m_sdksUpdatedConnection);
 
+    QString errors;
+
     QList<MerSdk*> lockedDownSdks;
     ok &= lockDownConnectionsOrCancelChangesThatNeedIt(&lockedDownSdks);
+    if (!ok)
+        errors.append("\nFailed lock down VM connections");
```

Comment:
All messages should be internationalized. In case of those that correspond to form values I would take the string from the label. After that using a list and join('\n') later will become more convenient.